crates/re_viewer/Cargo.toml

```diff
@@ -113,3 +113,9 @@ web-sys = { version = "0.3.52", features = ["Window"] }
 
 [build-dependencies]
 re_build_tools.workspace = true
+
+# External
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+serde_yaml = "=0.9.21"
```

Comment:
So, afaict this is to parse the markdown frontmatter, correct? Could we be using `toml` in there instead?

Reply:
Yes, we'd also have to update everything else that reads the frontmatter. Currently that's at least landing and `build_demo_app.py`

Reply:
Please do 🙏 , but can be saved for another PR